smib-fast.Dockerfile

```diff
@@ -24,7 +24,7 @@ WORKDIR /app
 COPY smib ./smib
 COPY pyproject.toml poetry.lock README.md ./
 
-RUN poetry install --without dev --no-root && rm -rf $POETRY_CACHE_DIR
+RUN poetry install --without dev && rm -rf $POETRY_CACHE_DIR
```

Comment:
This "installs" the smib package into the container - allowing the package version to be reliably extracted